Gulpfile.js

```diff
@@ -148,6 +148,7 @@ gulp.task('lint-client', function () {
         .src([
             'src/client/core/page-unload-barrier.js',
             'src/client/core/xhr-barrier.js',
+            'src/client/runner/automation/playback/upload.js',
```

Comment:
Please, replace this several rules with `src/client/runner/automation/**/*.js`
We have refactored almost all files there already